metabase/metabase
# Toggle series visibility from chart legend (pie charts)

Implements #2697 for pie charts ([product doc](https://www.notion.so/metabase/Let-users-hide-show-series-by-clicking-on-the-legend-90a5ec138b534f039136ea7f377252b8?pvs=4)) and effectively closes #2697

Allows to toggle series visibility from the chart legend by clicking the legend item dot (FWIW, series visibility set like that isn't meant to be persisted). Also, updates the `pie.rows` visualization setting widget to better distinguish its "hide" action from the new legend behavior (similarly to `graph.series_order` in cartesian charts)

### To verify

1. Create a multi-series chart (e.g., Count of Orders per Product Category)
2. Ensure you can click the legend dots to show/hide the slice
3. Ensure you can't hide the last slice
4. Ensure you can hide the "Other" slice
5. In chill mode, ensure you can manage slice order and related settings via the "Data" tab

### Demo

https://github.com/user-attachments/assets/b0a32d8c-0d3c-442a-98a4-40e973469dcb
